multiqc/utils/config_defaults.yaml

```diff
@@ -169,6 +169,7 @@ module_order:
     - 'rseqc'
     - 'busco'
     - 'goleft_indexcov'
+    - 'disambiguate'
```

Comment:
What is your reasoning for doing so?

Disambiguate works on two aligned bam files, and produces another filtered bam file. (It doesn't work with the fastq files.) The stats are for the filtered bam file, which is why I put it in post.